doc/changelog/12-misc/14041-cs_lambda.rst

```diff
@@ -0,0 +1,6 @@
+- **Added:**
+  Enable canonical `fun _ => _` projections,
+  see :ref:`canonicalstructures` for details.
+  (`#14041 <https://github.com/coq/coq/pull/14041>`_,
+  by Jan-Oliver Kaiser and Pierre Roux,
+  reviewed by Cyril Cohen and Enrico Tassi).
```

Comment:
I thought that the category for this kind of change should be `02-specification-language`.

Reply:
it makes more sense than misc indeed. Feel free to move this file to the appropriate directory in another PR (maybe the one about dependent CS)